### ISSUE TITLE
Multiple tree views:

### ISSUE THREAD

**Issue by Dexagod** [TREEcg/specification#30]
Addition to section for [metadata](https://treecg.github.io/specification/#metadata).

This first sentence makes it seem as if there is only a single tree view.
Maybe we can already allude to the point about [multiple views](https://treecg.github.io/specification/#multiple-views)

Maybe in the point about multiple views we should also indicate that these are multiple fragmentations of the collection (maybe in other wording)

**Comment by julianrojas87** [comment]
I was going to open a similar issue about this section. I would rephrase it as follows:

...a TREE client MUST **be able to** traverse all objects of tree:view linked to this particular collection...

**Comment by pietercolpaert** [comment]
New spec has been published
